iosHyperskillApp/iosHyperskillApp/Sources/Modules/Track/Views/DiscoverNext/TopicToDiscoverNextButtonView.swift

```diff
@@ -10,7 +10,7 @@ extension TopicToDiscoverNextButtonView {
         var buttonStyle = OutlineButtonStyle(borderColor: .border, alignment: .leading, paddingEdgeSet: [])
 
         let learnNextBadgeVerticalPadding: CGFloat = 4
-        let learnNextBadgeVerticalOffsetRatio = -0.25
+        let learnNextBadgeVerticalOffsetRatio = -0.125
```

Comment:
If the topic name will be longer -> badge will be outside of the widget (not centered)

<img width="449" alt="Screenshot 2023-02-16 at 5 51 12 PM" src="https://user-images.githubusercontent.com/8586063/219330691-96ee1e62-5a7a-4de5-962c-9bff18d87b7f.png">

You can add a view modifier to measure badge height from this [blog post](https://swiftuirecipes.com/blog/getting-size-of-a-view-in-swiftui).

Update badge rendering to this and all will be OK:
```swift
@ViewBuilder
private var learnNextBadge: some View {
    if isLearnNext {
        GeometryReader { geometry in
            Text(Strings.Track.TopicsToDiscoverNext.learnNextBadge)
                ...
                .measureSize { viewSize = $0 }
                .offset(
                    CGSize(
                        width: LayoutInsets.defaultInset,
                        height: -(viewSize.height / 2)
                    )
                )
        }
    }
}
```

<img width="473" alt="Screenshot 2023-02-16 at 5 55 03 PM" src="https://user-images.githubusercontent.com/8586063/219331581-efc2902c-8cd9-4b88-8d7e-f5378f26319c.png">